eZ/Publish/Core/MVC/Symfony/Controller/Content/DownloadController.php

```diff
@@ -63,7 +63,7 @@ public function downloadBinaryFileByIdAction(Request $request, int $contentId, i
     protected function findFieldInContent(int $fieldId, Content $content): Field
     {
         foreach ($content->getFields() as $field) {
-            if ($field->getId() === $fieldId) {
+            if ($field->id === $fieldId) {
```

Comment:
Normally I would request to add missing `getId` method, but given this is unmaintained 2.5 fix and it can go into 2.5 **only**, I'll allow it. Magic is bad because of performance (from 0.2s to 5s when it's scaled to e.g 30 000 calls, which is not uncommon on simple properties).
Here we can leave it as-is.